diff --git a/data.cpp b/data.cpp
index 4dfbde2..69ba098 100644
--- a/data.cpp
+++ b/data.cpp
@@ -725,21 +725,27 @@ void DataManager::SetDefaultValues()
 	mData.SetValue(TW_BACKUP_NAME, "(Auto Generate)");
 
 	mPersist.SetValue(TW_INSTALL_REBOOT_VAR, "0");
-	mPersist.SetValue(TW_SIGNED_ZIP_VERIFY_VAR, "0");
+	// mPersist.SetValue(TW_SIGNED_ZIP_VERIFY_VAR, "0");
+	mPersist.SetValue(TW_SIGNED_ZIP_VERIFY_VAR, "1");   // enfoce sign verification
 	mPersist.SetValue(TW_DISABLE_FREE_SPACE_VAR, "0");
 	mPersist.SetValue(TW_FORCE_DIGEST_CHECK_VAR, "0");
 	mPersist.SetValue(TW_USE_COMPRESSION_VAR, "0");
-	mPersist.SetValue(TW_TIME_ZONE_VAR, "CST6CDT,M3.2.0,M11.1.0");
+    // mPersist.SetValue(TW_TIME_ZONE_VAR, "CST6CDT,M3.2.0,M11.1.0");
+    // mPersist.SetValue(TW_TIME_ZONE_VAR, "JST-9JSTDT");
+    mPersist.SetValue(TW_TIME_ZONE_VAR, "JST-9");
 	mPersist.SetValue(TW_GUI_SORT_ORDER, "1");
 	mPersist.SetValue(TW_RM_RF_VAR, "0");
 	mPersist.SetValue(TW_SKIP_DIGEST_CHECK_VAR, "0");
 	mPersist.SetValue(TW_SKIP_DIGEST_GENERATE_VAR, "0");
 	mPersist.SetValue(TW_SDEXT_SIZE, "0");
 	mPersist.SetValue(TW_SWAP_SIZE, "0");
-	mPersist.SetValue(TW_SDPART_FILE_SYSTEM, "ext3");
-	mPersist.SetValue(TW_TIME_ZONE_GUISEL, "CST6;CDT,M3.2.0,M11.1.0");
+	// mPersist.SetValue(TW_SDPART_FILE_SYSTEM, "ext3");
+	mPersist.SetValue(TW_SDPART_FILE_SYSTEM, "ext4");
+    // mPersist.SetValue(TW_TIME_ZONE_GUISEL, "CST6;CDT,M3.2.0,M11.1.0");
+	mPersist.SetValue(TW_TIME_ZONE_GUISEL, "JST-9");
 	mPersist.SetValue(TW_TIME_ZONE_GUIOFFSET, "0");
-	mPersist.SetValue(TW_TIME_ZONE_GUIDST, "1");
+	// mPersist.SetValue(TW_TIME_ZONE_GUIDST, "1");
+	mPersist.SetValue(TW_TIME_ZONE_GUIDST, "0");
 	mData.SetValue(TW_ACTION_BUSY, "0");
 	mData.SetValue("tw_wipe_cache", "0");
 	mData.SetValue("tw_wipe_dalvik", "0");
@@ -890,8 +896,8 @@ void DataManager::SetDefaultValues()
 	mConst.SetValue("tw_oem_build", "1");
 #else
 	mConst.SetValue("tw_oem_build", "0");
-	mPersist.SetValue("tw_app_prompt", "1");
-	mPersist.SetValue("tw_app_install_system", "1");
+	mPersist.SetValue("tw_app_prompt", "0");
+	mPersist.SetValue("tw_app_install_system", "0");
 	mData.SetValue("tw_app_install_status", "0"); // 0 = no status, 1 = not installed, 2 = already installed
 #endif
 
